eng/common/TestResources/New-TestResources.ps1

```diff
@@ -393,6 +393,17 @@ foreach ($templateFile in $templateFiles) {
                 $DebugPreference = "Continue"
             }
             New-AzResourceGroupDeployment -Name $BaseName -ResourceGroupName $resourceGroup.ResourceGroupName -TemplateFile $templateFile -TemplateParameterObject $templateFileParameters
+        } catch {
+            $msg = @(
+                "",
+                "==================================================",
+                "For help debugging live test provisioning issues,",
+                "see http://aka.ms/azure-sdk-live-test-help",
```

Comment:
We have a common prefix of "azsdk" for most of the links for our team so I'd suggest you do something similar here. Something like "azsdk/engsys/live-test-help" also please add adpteam as an SG for the owner of the link.